audb/core/info.py

```diff
@@ -184,14 +184,19 @@ def header(
         *,
         version: str = None,
         cache_root: str = None,
-) -> audformat.Database:
+        timeout: float = -1,
+) -> typing.Optional[audformat.Database]:
```

Comment:
I guess your workflow idea is that if a user sets a timeout > 0, she is responsible herself to check that the return value is not `None` and only then try to access the data. An alternative would be to raise an error if a timeout occurs, but then the user would need to write a `try`-`except` statement. So I guess your solution makes sense.

Reply:
Yes, I would prefer to not raise an error.